episodes/episodes.Rmd

```diff
@@ -674,21 +674,21 @@ Table: example table with overflowing text in three columns
 
 :::::::::::::: solution
 
-To get a rougly 1:5:1 ratio, you can use two separators for the short columns
+To get a roughly 1:5:1 ratio, you can use two separators for the short columns
 and ten separators for the wide column:
 
 ```markdown
 Table: example table with overflowing text in three columns
 
 | first | second     | third |
-| --    | :--------: | --    |
+| --    | :----------: | --    |
```

Comment:
The `:` count as characters for length, so the 1:5:1 ratio was correct.

```suggestion
| --    | :--------: | --    |
```

The original produces an HTML table with these widths:

```html
<colgroup>
  <col width="14%">
  <col width="71%">
  <col width="14%">
</colgroup>
```

whereas this modification changes the ratios to be roughly 1:6:1.

```html
<colgroup>
  <col width="12%">
  <col width="75%">
  <col width="12%">
</colgroup>
```